suite-native/atoms/src/DiscreetText.tsx

```diff
@@ -16,9 +16,9 @@ type DiscreetTextProps = TextProps & {
 };
 
 // eslint-disable-next-line @typescript-eslint/no-var-requires
-const satoshiFont = require('../../../packages/theme/fonts/TTSatoshi-Regular.otf');
+const satoshiFont = require('../../../packages/theme/fonts/TTSatoshi-Medium.otf');
 
-const isDiscreetModeOn = atomWithUnecryptedStorage<boolean>('isDiscreetModeOn', false);
+const isDiscreetModeOn = atomWithUnecryptedStorage<boolean>('isDiscreetModeOn', true);
```

Comment:
This should be false by default?

Reply:
You are right. I had changed it for debugging and the forgot to change it back. fixed in 94ef45ffb.